src/sentry/incidents/logic.py

```diff
@@ -471,6 +481,21 @@ class AlertRuleNameAlreadyUsedError(Exception):
 }
 
 
+def get_alert_resolution(time_window: int, organization) -> int:
+    resolution = DEFAULT_ALERT_RULE_RESOLUTION
+
+    if features.has("organizations:metric-alert-load-shedding", organization=organization):
+        windows = sorted(DEFAULT_ALERT_RULE_LOAD_SHEDDING_RESOLUTIONS.keys())
+        index = bisect.bisect_right(windows, time_window)
```

Comment:
👀 if we're utilizing `index - 1`, does bisect_left work?

Reply:
Changing this to bisect_left would also change how the index lookup works;

For example:
```python
>>> import bisect
>>> bisect.bisect_left([1,2,3], 2)
1
>>> bisect.bisect_right([1,2,3], 2)
2
```

In this case, we'd want the functionality of bisect_right to get the correct result from the dictionary.